### PR TITLE
Capstone project module 1 Microverse

### PR DESCRIPTION
**Pull request by J-C-S-V**
Here is what I did on this milestone:

- Create 2 pages with HTML, one for the main page and the other for the about page
- Manipulate the DOM using JavaScript to create the speakers section
- Use JavaScript to add the menu on mobile view
- Use mobile first approach
- Style and personalize the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+          />
+        </defs>
+      </svg>
+      <img
+        class="header__img"
+        src="./img/logo.png"
+        alt="CC Global Summit 2015"
+      />
+    </section>
+    <section class="hero"></section>
```
Comment (nahnahdev) [inline index.html]: - [x] Great job so far @J-C-S-V :muscle: I noticed that the hero sections of your `home` page and `about` page are both currently images. While the images may look nice, I suggest building the hero sections using HTML and CSS. By doing so, you can improve the accessibility and SEO of your website, as well as provide more flexibility for future changes. Additionally, building the hero sections from scratch will allow you to showcase your coding skills and creativity. :100: 
- [x] So kindly build the hero sections for both your `home` page and your `about` page in both the `mobile` and `desktop` views and ensure that all the parts of the hero section in the original design are present in your design:+1: 
- [x] Lastly, Per the [Project Requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md), You should personalize the content of your page. 
       - Choose a topic that is different than the one in the original design and ensure that 
       -  All page elements have similar spacing, borders, shadows, and paddings to the original design. :+1: 
- Your About Page
image

- Your Home Page
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+          stroke="#A04038"
+          stroke-width="2"
+        />
+      </svg>
+      <div class="cards-container">
+        <article class="cards">
+          <img
+            class="cards__childs"
+            src="./img/program_icon.png"
+            alt="speaker"
```
Comment (nahnahdev) [inline index.html:130]: - [x] In the main part of your project, kindly ensure that the spacing between the elements is the same or similar to that of the original design as I mentioned earlier, as per the project requirements, you are to ensure that all page elements have similar spacing, borders, shadows, and paddings as the original design.
- [x] I will suggest that you close the spacing between the elements in the grid and increase the spacing below the `See the whole program` text  and increase the spacing above the `Main Program` heading :+1: You can refer to the images below :100:  
- Your Design
image

- The Original Design
image

File: index.css [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+  border: none;
+  padding: 1.5rem 3rem;
+  margin: 1rem 0;
+}
+
+.speakers {
+  display: flex;
+  gap: 15px;
+  padding: 4%;
+}
```
Comment (nahnahdev) [inline index.css:133]: - [x] Great job using Javascript to dynamically generate the speaker's section. However, I noticed that you're missing the short horizontal line that comes after the speaker's name and title and separates them both from the speaker's description  have attached images below for your reference :+1: 
 - [x] I also noticed that the width of each speaker when on a desktop screen is wider than that of the original design. Please try to ensure that the layout of your projects looks similar to that of the original design :100: 
 - Your Design
 
image

 - The Original Design
 
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+        <div class="article-wrapper">
+          <article class="main__article">
+            <h2 class="main__article-h2">2011</h2>
+            <p class="main__article-p">CC Global Summit 2011 in Warsaw</p>
+          </article>
+          <article class="main__article">
+            <h2 class="main__article-h2">2011</h2>
+            <p class="main__article-p">CC Global Summit 2011 in Warsaw</p>
+          </article>
+        </div>
```
Comment (nahnahdev) [inline about.html]: - [x] In this part of your about section, kindly ensure that the images you used are overlayed to look like the images used in the original design so that the text in the images will be more legible :100: 
- [x] KIndly increase the size of the images as well and ensure that their layout is similar to that of the original design :+1: 
- Your Design
image

- The Original Design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+      <ul class="navbar__ul">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="li-special"><a href="#">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <!-- Desktop view end -->
```
Comment (nahnahdev) [inline index.html:46]: - [x] In your desktop view, the logo in your `nav` should redirect to the `homepage` when clicked but that feature hasn't been implemented yet so kindly include that :100: 
- [x] In your mobile view, I noticed that your mobile menu only shows up after 630px. However, it's important to ensure that it is functional and shows up on a variety of mobile devices. I suggest adjusting the CSS media query for your mobile menu to trigger at a lower breakpoint, such as 480px or 320px. This will allow users on smaller devices to easily access the menu and improve the overall user experience. I have attached a video below for your reference :100: 
- Your Design 
[Screencast from 03-18-2023 12:14:57 AM.webm](https://user-images.githubusercontent.com/102699958/226072632-e7452197-ef1d-45a2-a07b-47fdb92d49ec.webm)

- [x] Also, I noticed that when the hamburger menu appeared, when it is clicked, the hamburger icon stays there and the close button is placed at the top of the nav items instead. I will suggest that you make the hamburger icon disappear when the menu is opened and the close button replaces the hamburger icon in the same position so it doesn't appear at the top of your hamburger menu that way, it improves the user's experience and make the menu more intuitive to use:wink: 
- [x] Kindly ensure that the background of the mobile menu is transparent or the same color/pattern as the top section background of your project as per the project requirements :+1:  which in this case is white :100: 
So kindly change the color of the background of your mobile menu to white or transparent :wink: 

- Your Design Now
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @J-C-S-V, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good use of grid and flexbox :heavy_check_mark: 
- Project deployed :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Required Changes ♻️
- [x] Great job recording the video walkthrough of your project it was informative and demonstrated your project very well. However, you didn't mention a piece of code or feature that you built that you are proud of so kindly record a short video showing this and add the link of the video to the pull request or to your README file before resubmitting your project for review :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+          <h3 class="main__h3 cards__childs">CC Ignite</h3>
+          <p class="cards__childs">
+            get opportunities to network with CC affiliates around the world,
+            also after the summit.
+          </p>
+        </article>
+      </div>
+      <button class="main__button">Join CC Global Summit 2015</button>
+
+      <p class="desktop-p">SEE THE WHOLE PROGRAM</p>
```
Comment (divinecharlotte) [inline index.html:189]: - [x] as requested. by the previous reviewer kindly consider increasing the padding-bootom of this paragraph so as to meet the provided design

File: index.css [hunk 7]
```diff
@@ -0,0 +1,366 @@
...
```
Comment (divinecharlotte) [inline index.css:133]: - [x] I can see that you tried to implement the required changes by the previous reviewer good job so far, but still there is some required changes you are missing out kindly consider checking on this [required change](https://github.com/J-C-S-V/The-Ethics-of-AI/pull/4#discussion_r1140840470) again and make sure you implement it before submitting for another review

File: index.js [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+      <p class="speakers__p-1">
+        CEO of Creativve Commons, ex COO of the Mozilla Foundation
+      </p>
+      <p class="speakers__p-2">
+        Ryan had been leading open-source projects at the Mozilla
+        Foundation such as the open-source move-ment
+      </p>
+    </aside>
+  </article>
+</div>`;
```
Comment (divinecharlotte) [inline index.js]: - [x] good job adding this section dynamically 👍 but you are still missing something here kindly consider assigning The list of speakers used to generate the HTML i to a variable as an array or object.
here is a hint 👇 
```
// You can use an array to store the objects
const speakerList = [ { name: "Speaker Name",... } ]
// It's ok to use an object too
const speakerList = { speaker1: { name: "Speaker Name",... } }
```

- [x]  and please put in consideration that The list of speakers contain a speaker object for each that holds their specific data.
- here is a hint too 👇 
```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```

File: about.html [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+        <div class="article-wrapper">
+          <article class="main__article">
+            <h2 class="main__article-h2">2011</h2>
+            <p class="main__article-p">AI conference 2011 in Buenos Aires</p>
+          </article>
+          <article class="main__article2">
+            <h2 class="main__article-h2">2011</h2>
+            <p class="main__article-p">AI conference 2011 in Warsaw</p>
+          </article>
+        </div>
```
Comment (divinecharlotte) [inline about.html]: - [x] as requested by the previous reviewer kindly consider adding an overlay on these images and give it the background color as given in the design and then after consider adding the text on it as well cause currently you have only a picture and some text on it  why this change! it will demonstrate you Css skills and also help you to meet the design requirements

File: index.css (hunk 7 above)
Comment (divinecharlotte) [inline index.css:133]: - [ ] i can see that you tried to implement some required changes good job so far 👍  but still there is some thing that you are missing out as requested by the previous reviewer kindly consider adding the short horizontal line that comes after the speaker's name and title and separates them both from the speaker's 
- [ ] I also noticed that the width of each speaker when on a desktop screen is wider than that of the original design. Please try to ensure that the layout of your projects looks similar to that of the original design 💯

File: index.html [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+      <img class="navbar-logo" src="./img/logo.png" alt="logo" />
+      <ul class="navbar__ul">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="li-special"><a href="#">CC Campaign</a></li>
+      </ul>
+    </nav>
```
Comment (divinecharlotte) [inline index.html:45]: - [x] kindly consider displaying the hamburger menu on the home section on the mobile version which will help to move from one page to another as you did on about page  and consider not displaying the log on mobile version
- your design �
Screen Shot 2023-03-18 at 08 10 05
� 


- provided design 👇 
Screen Shot 2023-03-18 at 08 11 34

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @J-C-S-V ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100:
- No linters errors :heavy_check_mark:
- Descriptive commit messages :heavy_check_mark:
- clean code :heavy_check_mark:
- Descriptive PR :heavy_check_mark:



## Required Changes :recycle:

- [x] Project documentation

- kindly consider adding  a clear and objective description about the project  in fact consider making  The topic and purpose of the project clear.  have to personalize your  content and this will help us understand well what you worked on 
- kindly consider adding a highlight about a piece of the code that you found interesting or you are proud of making.in order to full-fill the below requirement 
> The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

> 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 10 above)
Comment (J-C-S-V) [reply index.html:45]: Hello @divinecharlotte,

I don't get what you're saying. Can you kindly check the dev tools on your side?. Mine works properly, also on my mobile phone.

menu fixed

Comment (tufoinnkuo10) [inline index.html:45]: - [x] please, like the previous reviewer noted, kindly ensure and check well that the hamburger menu is visible in the home page of the mobile version as it helps to move from one page to another as you did on about page. kindly check if you pushed all your code. 
- [x] also, the previous reviewer noted that you should consider not displaying the logo on mobile version. Remember it is someone's design, and you were to personalize the content but follow the design. Please, following the design guidelines is always important. 
demo33

File: index.html [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+        <img
+          src="./img/partners_daumkakaologo 1.png"
+          alt="daumkakaologo logo"
+        />
+        <img src="./img/partners_googlekorea.png" alt="google logo" />
+        <img src="./img/partners_mozilla.png" alt="mozilla logo" />
+        <img src="./img/partners_naver.png" alt="naver logo" />
+      </div>
+    </section>
+    <footer class="about_footer">
```
Comment (tufoinnkuo10) [inline index.html:223]: - [optional] kindly reduce the logo size in the footer, so it can align with the original design
Screenshot (885)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @J-C-S-V 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design ok

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 12]
```diff
@@ -0,0 +1,237 @@
...
+          />
+        </defs>
+      </svg>
+      <a href="index.html">
+        <img
+          class="header__img"
+          src="./img/logo.png"
+          alt="CC Global Summit 2015"
+        />
```
Comment (amejid) [inline index.html:82]: - [x] Great job adding the hamburger on the homepage. :+1: Kindly remove the logo image on the mobile version as suggested by the previous reviewers. ✍️


| Your design                                                             | Behance design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

File: index.html [hunk 13]
```diff
@@ -0,0 +1,237 @@
...
+        <img src="./img/partners_mozilla.png" alt="mozilla logo" />
+        <img src="./img/partners_naver.png" alt="naver logo" />
+      </div>
+    </section>
+    <footer class="about_footer">
+      <img
+        class="about_footer-img"
+        src="./img/about_footer.png"
+        alt="Compnay name"
+      />
```
Comment (amejid) [inline index.html:228]: - [x] [OPTIONAL] Your footer image looks huge when compared to the template. I highly suggest you reduce the size. 😎️
This is how it is currently looking.

Screenshot from 2023-03-18 19-32-08

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @J-C-S-V, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 12 above)
Comment (J-C-S-V) [reply index.html:82]: Hello @amejid,

Could you please double check the figma's design? That feature seem to be missing.  The logo should be present in both Home and About sections for mobile

image

**Review by jerryowusu — APPROVED** [review]
Hi @J-C-S-V , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Great design :heavy_check_mark: 
No linter errors :heavy_check_mark: 

## Optional suggestions

- [x] Great design and nice project. However, I am surprised as to how you got the design with the logo at the right side of the hamburger menu from since the original design shown below does not have the logo. I am taking a huge risk in approving your project since I do not want you to repeat the whole week due to this 'trivial' issue. Kindly consider adhering to the advice of the code reviewers in subsequent projects in order to avoid this back and forth. :pray: 


Screenshot from 2023-03-18 22-24-27


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J-C-S-V** [comment]
Thank you @jerryowusu,

Your explanation was much clearer. Now I understand what's the issue, I just followed the figma's design I was given, that's where the discrepancy is. It seems that Microverse had 2 different templates.

I'll keep that minimun change in mind since that's an easy fix.

Reagards!
